toolchain/install/install_filegroups.bzl

```diff
@@ -105,36 +90,21 @@ def make_install_filegroups(name, no_driver_name, pkg_name, install_dirs, prefix
 
             if "target" in entry:
                 if entry["executable"]:
-                    symlink_file(
-                        name = prefixed_path,
-                        symlink_binary = entry["target"],
-                    )
                     mode = "0755"
                 else:
-                    symlink_file(
-                        name = prefixed_path,
-                        symlink_label = entry["target"],
-                    )
                     mode = "0644"
+                copy_file(
+                    name = prefixed_path + ".copy",
+                    src = entry["target"],
+                    out = prefixed_path,
+                    is_executable = entry["executable"],
+                )
```

Comment:
To be sure, I'd previously looked at `copy_file` and the [default is to create a symlink](https://github.com/bazelbuild/bazel-skylib/blob/main/rules/private/copy_file_private.bzl#L91). I think the symlink is actually why this works -- it's symlinks the busybox is happy to work with. But that's always why this is essentially a rollback of #4505.

That said, I was avoiding non-symlink copies because we'd need to do it per-busyboxed binary and have been running into GitHub action runner disk space limits. LLVM's current busybox setup supports 43 binaries, so that's the scale I'd want to design for.

Reply:
Arg, I was reading a different document that had the opposite default.

To be clear, my suggestion is to use `copy_file` *without* allowing a symlink for the busybox itself, or as I mentioned in the PR description, causing the busybox to be directly created in the install tree so that it isn't a symlink.

Anyways, following up on the high-level issue above.